### ISSUE TITLE
no 'path' filed for portable notepad++

### ISSUE THREAD

**Issue by teodias** [gitextensions/gitextensions#10013]
### Feature description

there is no path lineedit for settings/git/config/editor
I can select notepad++ but cannot enter any path for the portable package.
I assume it will try to find it by any registry key?

### Environment

- GitExtensions version: GitExtensions-Portable-3.5.4.12724-65f01f399.zip
- GIT version: GitExtensions-Portable-3.3.1.7897.zip
- OS version: Microsoft Windows [Version 10.0.19044.1706]
- .NET version: how do i check?

**Comment by vbjay** [comment]
Tap help then about. Click copy button.

**Comment by vbjay** [comment]
No. It will not. Portable still expects a path.   You can attempt a relative path frome get folder to notepad ++ folder.

**Comment by vbjay** [comment]
Oh right the editor used to have a browse button.  Now it detects paths.

**Comment by vbjay** [comment]
Pick one of the values and modify the  right .gitconfig file. 
![image](https://user-images.githubusercontent.com/3629489/170624653-ef39c5d6-4328-487a-a7e2-c9dd4955c7b9.png)
![image](https://user-images.githubusercontent.com/3629489/170625131-bfd67fa2-3726-4f9c-85c3-cf6243808463.png)
![image](https://user-images.githubusercontent.com/3629489/170625233-e246737c-5875-420e-a991-08ec78aab52d.png)
![image](https://user-images.githubusercontent.com/3629489/170625462-f2f1c966-00d3-4d2e-92f5-dad117d29d0c.png)
```
"../../Notepad++Portable\App\Notepad++64\notepad++.exe" -multiInst -nosession
```
the dropdown is editable and you can type whatever you want

**Comment by RussKie** [comment]
Can you please clarify what's the ask here?

**Comment by gerhardol** [comment]
![image](https://user-images.githubusercontent.com/6248932/171484269-bef9d1ce-1fa6-4c15-856d-afbb656b3af1.png)

The default dropdown adds some common editors, that normally is added to your PATH if you have istalled the apps.
You may enter any path too.

To have a Browse button is a little tricky if the default arguments are to be added.
Get the path outside GE.

I suggest no functionality change here.
